NEWS.md

```diff
@@ -1,12 +1,14 @@
 ## rtables.officer 0.0.2.9003
 
-### New Features
  * Added option to flush pages when exporting to `.docx`. It is possible to put tables on separate pages by doing `export_as_docx(add_page_break = TRUE)`.
-
+ * Now it is possible to transform and export also `rlistings` objects.
```

Comment:
@shajoezhu I think this deserves a release. There are also major fixes in the draft pagination system